src/main/webapp/app/exercises/shared/team/team-students-list/team-students-list.component.html

```diff
@@ -1,6 +1,13 @@
 <ul class="list-group list-group-horizontal student-group">
-    <li class="list-group-item student-group-item" *ngFor="let student of students" [class.has-error]="errorStudentLogins.includes(student.login)">
-        <a *ngIf="renderLinks" [routerLink]="['/admin', 'user-management', student.login]">{{ student.name }} ({{ student.login }})</a>
-        <span *ngIf="!renderLinks">{{ student.name }} ({{ student.login }})</span>
+    <li
+        class="list-group-item student-group-item"
+        *ngFor="let student of students"
+        [class.has-error]="errorStudentLogins.includes(student.login) || (withRegistrationNumber && errorStudentRegistrationNumbers.includes(student.visibleRegistrationNumber))"
```

Comment:
This is a huge expression. Would it be better to create a variable and save the state there?